<!-- received="Mon Jan 13 07:55:52 1997 MDT" -->
<!-- sent="Mon, 13 Jan 1997 15:05:41 +0100 (MET)" -->
<!-- name="Eugene Leitl" -->
<!-- email="Eugene.Leitl@lrz.uni-muenchen.de" -->
<!-- subject="SCI:ALIFE: the things of shape to come" -->
<!-- id="Pine.SOL.3.91.970113145857.1412A-100000@sun6" -->
<!-- inreplyto="" -->
<title>extropians: SCI:ALIFE: the things of shape to come</title>
<h1>SCI:ALIFE: the things of shape to come</h1>
Eugene Leitl (<i>Eugene.Leitl@lrz.uni-muenchen.de</i>)<br>
<i>Mon, 13 Jan 1997 15:05:41 +0100 (MET)</i>
<p>
<ul>
<li> <b>Messages sorted by:</b> <a href="date.html#582">[ date ]</a><a href="index.html#582">[ thread ]</a><a href="subject.html#582">[ subject ]</a><a href="author.html#582">[ author ]</a>
<!-- next="start" -->
<li> <b>Next message:</b> <a href="0583.html">Natasha V. Mor: "Re: &gt;H Art: Exemplar Exhibition Up"</a>
<li> <b>Previous message:</b> <a href="0581.html">Eugene Leitl: "SCI:BIO:genome length and organism complexity"</a>
<!-- nextthread="start" -->
<li> <b>Next in thread:</b> <a href="0584.html">Eugene Leitl: "SCI:ALIFE: the walk is not random, nor is it a walk"</a>
<li> <b>Reply:</b> <a href="0584.html">Eugene Leitl: "SCI:ALIFE: the walk is not random, nor is it a walk"</a>
<!-- reply="end" -->
</ul>
<hr>
<!-- body="start" -->
"Extended Molecular Evolutionary Biology: Artificial Life Bridging the Gap<br>
Between Chemistry and Biology", P. Schuster, in "Artificial Life, an<br>
Overview", Christopher G. Langton (ed.), MIT Press (1995), pp. 39-60.<br>
<p>
[...] Evolutionary phenomena, in particular selection and adaptation to<br>
changes in the environment, occur only at conditions far away from the<br>
thermodynamic equilibrium.  Spiegelman studied RNA molecules from small<br>
bacteriophages and created nonequilibrium conditions by means of the<br>
serial transfer technique (Figure 1). Material consumed by multiplication<br>
of RNA molecules is renewed, and the degradation products are removed at<br>
the end of constant time intervals by transfer of small samples into an<br>
excess of fresh stock solution. Continuous renewal and removal can be<br>
achieved in elaborate flow reactors [3,4]. [...] As far as the principle<br>
of polynucleotide replication is concerned, there seems to be no reason<br>
why molecular replication should need a protein catalyst. Extensive<br>
studies by Orgel and coworkers [...] have indeed shown that<br>
template-induced synthesis of complementary strands of RNA can be<br>
achieved under suitable conditions without an enzyme. [...] It seems<br>
necessary to stress a fact that is often overlooked or even ignored by<br>
theorists and epistemologists. Molecular replication is anything but a<br>
trivially occuring function of the molecules. [...] Everybody who has<br>
experience with primitive computing machines knows that the copy<br>
instruction is a very simple function. Chemistry and early biological<br>
evolution are radically different from computer science in this respect:<br>
Replication has to find a simultaneous solution to all requirements,<br>
which is generally in conflict with common physical chemistry. Working<br>
compromises between contradicting demands are rare, and, hence, only<br>
highly elaborate structures might be able to replicate efficiently<br>
without specific help. [...] The stationary mutant distribution is<br>
characterized as _quasispecies_ [...] because it represents the genetic<br>
reservoir of asexually replicating populations. An increase in error rate<br>
in the replication on a given fitness landscape leads to a broader<br>
spectrum of mutants and, thus, makes evolutionary optimization faster and<br>
more efficient in the sense that populations are less likely caught in<br>
local fitness optima. There is, however, a critical error threshold<br>
[...]: If the error rate exceeds the critical limit, heredity breaks<br>
down, populations are drifting in the senes that new RNA sequences are<br>
formed steadily, old ones disappear, and no evolutionary optimization<br>
according to Darwin's principle is possible [...]. Experimental analysis<br>
of several RNA virus populations has shown that almost all chain lengths<br>
are adjusted to yield error rates close to the threshold value. Thus, RNA<br>
viruses appear to adapt to their environment by driving optimization<br>
efficiency towards the maximum. [...] Evolutionary optimization is viewed<br>
appropriately as an adaptive walk on a rugged fitness landscape [...].<br>
Population dynamics on realistic landscapes based on RNA folding has been<br>
studied by computer simulations [...]. Error thresholds were detected on<br>
these rather very rugged landscapes, too. On completely flat fitness<br>
landscapes, and new phenomenon was observed [...]: At sufficiently high<br>
replication accuracy populations move as coherent peaks in sequence<br>
space. There is, however, again a critical error rate. If it is exceeded,<br>
the population loses its coherence in sequence space and becomes<br>
disperse. It is suggestive, therefore, to call this second critical error<br>
rate the _dispersion threshold_. [...]<br>
<p>
RNA secondary structures are first approximations to the spatial<br>
structures of RNA molecules. They are understood as listings of the<br>
Watson-Crick-type base pairs in the actual structure and may be<br>
represented as planar graphs [...]. We consider RNA secondary structures<br>
as elements of an abstract _shape space_. As in the case of sequences<br>
(where the Hamming distance d_h represents a metric for the sequence<br>
space), a measure of relationship of RNA structures can be found that<br>
induces a metric on the shape space. We derived this distance measure<br>
from trees that are equivalent to the structure graphs, and accordingly<br>
it is called a tree distance, d_t. Thus, RNA folding can be understood as<br>
a mapping from one metric space into another, in particular, from<br>
sequence space into shape space. A path in sequence space corresponds<br>
uniquely to a path in shape space. (The inversion of this statement,<br>
however, is not true as we shall mention in the section 4.) [...]<br>
<p>
The sequence space is a bizarre object: It is of very high dimension<br>
(because every nucleotide can be mutated independently, its dimension<br>
coincides with the chain lenght of RNA: 25 &lt; n &lt; 500 for RNA in test tube<br>
experiments, 250 &lt; n &lt; 400 for viroids, and 3500 &lt; n &lt; 20000 for (most)<br>
RNA viruses), but there are only a few points on each coordinate axis<br>
(\kappa points; \kappa is the number of digits in the alphabet: \kappa=2<br>
for AU and GC, \kappa=4 for AUCG). The number of secondary structures<br>
that are acceptable as minimum free energy structures of RNA molecules is<br>
much smaller than the number of different sequences and can be estimated<br>
by means of proper combinatorics [...]: In case of natural (AUCG)<br>
molecules we have about 1.485 x n^{-3/2} (1.849)^n structures for 4^n<br>
sequences. The mapping from sequence space into shape space is not<br>
invertible: Many sequences fold into the same secondary structure. We<br>
cannot expect that our intuition, which is well trained with mostly<br>
invertible maps in three-dimensional space, will guide us well through<br>
sequence and shape spaces. [...] As indicated [...], we search for<br>
_neutral paths_ through sequence space. The Hamming distance from the<br>
reference increases monotonously along such a neutral path, but the<br>
structure remains unchanged. A neutral path ends when no further neutral<br>
sequence is found in the neighbourhood of the last sequence. [...]<br>
Clearly, a neutral path cannot be longer than the chain length [...]. It<br>
is remarkable that about 20% of the neutral paths have the maximum length<br>
and lead through the whole sequence space to one of the sequences that<br>
differ in all positions from the reference, but have the same structure.<br>
<p>
Combination of information derived from [...] provides insight into the<br>
structure of the shape space of RNA secondary structures, which is basic<br>
to optimization of RNA molecules already in an RNA world. Our results can<br>
be summarised in four statements:<br>
<p>
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!<br>
<p>
1. Sequences folding into one and the same structure are distributed<br>
   randomly in sequence space.<br>
<p>
2. The frequency distribution of structures is sharply peaked. (There are<br>
   many comparatively few common structures and many rare ones.)<br>
<p>
3. Sequences folding into all common structures are found within<br>
   (relatively) small neighbourhoods of any random sequence.<br>
<p>
4. The shape space contains extended neutral networks joining sequences<br>
   with identical structures. (A large fraction of neutral paths leads<br>
   from the initial sequence through the entire sequence space to a final<br>
   sequence on the opposite side -- there are (\kappa - 1)^n sequences<br>
   that differ in all positions from an initial sequence).<br>
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!<br>
<p>
[...] These results suggest straightforward strategies in the search for<br>
new RNA structures. It provides little advantage to start from natural or<br>
other preselected sequences because any random sequence would do equally<br>
well as the starting molecules for the selection cycles of evolutionary<br>
biotechnology shown in [...]. Any common secondary structure with optimal<br>
functions is accessible in a few selection cycles. [...] If no RNA<br>
molecule with satisfactory properties is found, a change to high error<br>
rate is adequate. Then the population spreads along the neutral network<br>
to other regions in sequence space, which can be explored in detail after<br>
tuning the error rate low again.<br>
<p>
The structure of shape space is highly relevant for evolutionary<br>
optimization in nature too. Because long neutral paths are common,<br>
populations drift readily through sequence space whenever selection<br>
constraints are absent. This is precisely what is predicted for higher<br>
organisms by the neutral theory of evolution [...] and what is observed<br>
in molecular phylogeny by sequence comparisons of different species. The<br>
structure of shape space provides also a rigorous answer to the old<br>
probability argument against the possiblity of successful adaptive<br>
evolution [...].<br>
<p>
'gene:<br>
<!-- body="end" -->
<hr>
<p>
<ul>
<!-- next="start" -->
<li> <b>Next message:</b> <a href="0583.html">Natasha V. Mor: "Re: &gt;H Art: Exemplar Exhibition Up"</a>
<li> <b>Previous message:</b> <a href="0581.html">Eugene Leitl: "SCI:BIO:genome length and organism complexity"</a>
<!-- nextthread="start" -->
<li> <b>Next in thread:</b> <a href="0584.html">Eugene Leitl: "SCI:ALIFE: the walk is not random, nor is it a walk"</a>
<li> <b>Reply:</b> <a href="0584.html">Eugene Leitl: "SCI:ALIFE: the walk is not random, nor is it a walk"</a>
<!-- reply="end" -->
</ul>
