<!-- received="Tue Aug  3 04:29:00 1999 MDT" -->
<!-- sent="Tue, 3 Aug 1999 03:28:52 -0700 (PDT)" -->
<!-- name="Robert J. Bradbury" -->
<!-- email="bradbury@www.aeiveos.com" -->
<!-- subject="Re: Alpha Radiation (fwd)" -->
<!-- id="Pine.SV4.3.91.990803022059.27795C-100000@www.aeiveos.com" -->
<!-- inreplyto="Alpha Radiation (fwd)" -->
<!-- version=1.10, linesinbody=118 -->
<html><head><title>extropians: Re: Alpha Radiation (fwd)</title>
<meta name=author content="Robert J. Bradbury">
<link rel=author rev=made href="mailto:bradbury@www.aeiveos.com" title ="Robert J. Bradbury">
</head><body>
<h1>Re: Alpha Radiation (fwd)</h1>
Robert J. Bradbury (<i>bradbury@www.aeiveos.com</i>)<br>
<i>Tue, 3 Aug 1999 03:28:52 -0700 (PDT)</i>
<p>
<ul>
<li> <b>Messages sorted by:</b> <a href="date.html#1450">[ date ]</a><a href="index.html#1450">[ thread ]</a><a href="subject.html#1450">[ subject ]</a><a href="author.html#1450">[ author ]</a>
<!-- next="start" -->
<li><a href="1451.html">[ Next ]</a><a href="1449.html">[ Previous ]</a>
<b>In reply to:</b> <a href="1439.html">John Clark</a>
<!-- nextthread="start" -->
</ul>
<!-- body="start" -->

<p>
<a href="1270.html#1450qlink1">&gt; John Clark &lt;jonkc@worldnet.att.net&gt; wrote:</a><br>

<p>
<a href="1439.html#1450qlink2">&gt; In case you hadn't noticed Alpern is talking about cancer, "oncogenic transformation"</a><br>
<i>&gt; just sounds better in a big expensive scientific book.</i><br>
<i>&gt; </i><br>
<i>&gt;     &gt;Only a *very* energetic particle has enough energy to interact with the DNA</i><br>
<i>&gt;     &gt;backbone and produce a double strand break.</i><br>
<i>&gt; </i><br>
<i>&gt; Nonsense. In the first place the backbone is not important, it's a break in the rungs</i><br>
<i>&gt; of the DNA ladder that's important because that's where all the digital information</i><br>
<i>&gt; is stored.</i><br>

<p>
John, this simply isn't true.  I would suggest that you go to PubMed, and lookup
a REVIEW article on "poly-ADP-ribose polymerase", "double-strand-breaks" and
"apoptosis" (or some combination thereof).

<p>
There are many *very* distinct pathways of mutation, repair and cell death.
I believe I may have glossed over the pathways involved with base mutations
vs. DSB.  Producing base mutations involves breaking a H-C bond on a base
[in most cases] (low energy).  Producing a double strand break requires breaking
the bonds on *both* DNA backbones (higher energy).  To produce a DSB you have to
deposit enough energy in a region covered by the distance between the DNA strands
to break both of them in a time that is less than the time required for single-strand
break repair to occur.  Since DSB require breaking two bonds in an volume that is
much bigger than that of a single base it is more difficult than a single base
mutation.

<p>
If you have enough double strand breaks you activate poly-ADP-ribose polymerase
which consumes NAD(P)H producing a "sugar-matrix" (if my recollection serves me
correctly) that tries to physically "lock" the DNA in place so the double strand
breaks don't flop around the entire nucleus in such a way that makes them difficult
to rejoin.  However if you deplete all of the NAD(P)H in the cell, you have
eliminated all of the reducing equivalents which makes oxidative defense
difficult.  The cells seem to have a threshold of tolerance for DSB.  If you
exceed that level (~5 in mammalian cells), then the cell activates the apoptosis
(cell death) program (through the p53 pathway I believe).

<p>
DSB are very important.  There is a specific kind of cancer (Burkitt's Lymphoma
I believe) that is the result of two different chomosomes getting broken
and put back together improperly so an improper upstream promoter sequence
activates a gene that promotes cancer.  Since the more DSB you have, the
greater the probability of mis-joining chromosomes is, the best bet is
to "fall on your sword" when that situation occurs.

<p>
Now, all of this is *much* different from the production of hydroxyl radicals
that attack the DNA bases, that *may* or *may* not be copied incorrectly.
Current thinking is that there are 3-4 different error repair pathways that
use 4+ different DNA polymerases in mammalian cells that each have different
abilities to copy DNA with/without various mutations accurately or inaccurately.
Evolution has presumably tried to evolve the relative activations and probabilities 
for these paths to serve its own goals (tradeoffs between repair and reproduction
and a mutation rate sufficient to drive evolution).

<p>
There are 5-8 different human genetic diseases (with large numbers of variants
[complementation groups] known) involved in defective DNA repair pathways.
These pathways involve at least 30-50 genes.  These genes don't exist
simply because nature is showing off.  They exist because there are many
types of damage that can occur and they have to be responded to in different
ways at different times.  Repair is an appropriate response if the damage
isn't too severe.  Cell death is an appropriate response if you think
the damage might lead to cancer.  Copying DNA (with mutations) is an appropriate
response if the cell is "essential".

<p>
For example, there is a specific mutation - a thymine dimer - that occurs
when UV radiation bonds breaks the T-A bonds across the DNA strands and
joins two adjacent T-T on the same backbone to each other.  There is
a specific enzyme DNA photolyase that utilizes low energy light to correct
this error.  There is a specific DNA polymerase (DNA Pol zeta) that
conveniently enough appears to be able to copy through T-T dimers
correctly inserting the A's on the opposite strand (while other DNA
polymerases halt or mis-copy).  Alpha particles or X-rays *don't*
produce thymine dimers (relative to the numbers that UV radiation does).

<p>
The main point of your comments that I am stuck on is your claim 
that alpha particles (of various energies) produce mutations that must
kill or mutate cells.  Given the different energies and the multitude
of materials that can be "hit" and the various response pathways that
can be invoked, I think that statement is a gross oversimplification.

<p>
<a href="1439.html#1450qlink3">&gt; In the second place, the chemical bonds in the rungs are much weaker than those in</a><br>
<i>&gt; the backbone, although even the strongest chemical bonds are of only a few electron</i><br>
<i>&gt; volts .</i><br>
<i>&gt;</i><br>
<i>&gt; I'm talking about average run of the mill radiation particles and they have thousands,</i><br>
<i>&gt; often millions, and sometimes many billions of electron volts of energy.</i><br>

<p>
Ok, now we are talking hard numbers.  So with all this excess energy, *most* of
it will be transfered to the water molecules (since they are the most abundant).
Those produce hydroxyl radicals (by knocking off a H atom).  The question then
becomes what is the probability of interaction of the hydroxyl radical with
(a) a DNA base, and (b) the atoms in the DNA backbone.  I can't answer the
question right now (since I've got to leave for the Bioastronomy conf. in
a few hours), but if you are really interested, remind me in a week or so
and I'll try to tease the numbers out of my sources.

<p>
Since the "data" is that high-LET radiation is more likely to produce mutations
than DSB (and cell death), I would presume that the hydroxyl radicals have
a higher probability of attacking a base than the backbone.  Low-LET radiation
may have completely different interaction probabilities.

<p>
<a href="1439.html#1450qlink4">&gt; Hey, don't tell me how smart you are, show me</a><br>

<p>
I can only try.
<br>
<i>&gt; </i><br>
<a href="1439.html#1450qlink5">&gt; I will too, should that unlikely event ever happen.</a><br>
<i>&gt;</i><br>
Grin...
<br>
<i>&gt; </i><br>
<a href="1439.html#1450qlink6">&gt; You're welcome to try but you'll have to do better than quoting long passages from a</a><br>
<i>&gt; book you don't understand.</i><br>
<i>&gt; </i><br>
I understand the physics in the books reasonably well, and the biological systems
involved better than anyone but a handful of people.  To really answer this question
properly requires a simulation at the atomic level.  There are programs that do this
(at LLNL &amp; LANL), but I doubt we can have access to them :-).

<p>
Robert
<!-- body="end" -->
<p>
<ul>
<!-- next="start" -->
<li><a href="1451.html">[ Next ]</a><a href="1449.html">[ Previous ]</a>
<b>In reply to:</b> <a href="1439.html">John Clark</a>
<!-- nextthread="start" -->
</ul>
</body></html>
