<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN"
                      "http://www.w3.org/TR/html4/strict.dtd">
<html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=US-ASCII">
<meta name="generator" content="hypermail 2.1.5, see http://www.hypermail.org/">
<title>Extropians: Re: Questions about human genome project</title>
<meta name="Author" content="Robert J. Bradbury (bradbury@aeiveos.com)">
<meta name="Subject" content="Re: Questions about human genome project">
<meta name="Date" content="2003-01-24">
<style type="text/css">
body {color: black; background: #ffffff}
h1.center {text-align: center}
div.center {text-align: center}
</style>
</head>
<body>
<h1>Re: Questions about human genome project</h1>
<!-- received="Fri Jan 24 21:52:03 2003" -->
<!-- isoreceived="20030125045203" -->
<!-- sent="Fri, 24 Jan 2003 20:52:01 -0800 (PST)" -->
<!-- isosent="20030125045201" -->
<!-- name="Robert J. Bradbury" -->
<!-- email="bradbury@aeiveos.com" -->
<!-- subject="Re: Questions about human genome project" -->
<!-- id="Pine.LNX.4.44.0301241921340.18822-100000@server.aeiveos.com" -->
<!-- charset="US-ASCII" -->
<!-- inreplyto="F58jl7tMmXm4yp4nbuk00009549@hotmail.com" -->
<!-- expires="-1" -->
<p>
<strong>From:</strong> Robert J. Bradbury (<a href="mailto:bradbury@aeiveos.com?Subject=Re:%20Questions%20about%20human%20genome%20project"><em>bradbury@aeiveos.com</em></a>)<br>
<strong>Date:</strong> Fri Jan 24 2003 - 21:52:01 MST
</p>
<!-- next="start" -->
<ul>
<li><strong>Next message:</strong> <a href="1378.html">Lee Corbin: "RE: Euro Trash Talk,"</a>
<li><strong>Previous message:</strong> <a href="1376.html">spike66: "sports"</a>
<li><strong>In reply to:</strong> <a href="1373.html">Nathanael Allison: "Questions about human genome project"</a>
<!-- nextthread="start" -->
<!-- reply="end" -->
<li><strong>Messages sorted by:</strong> 
<a href="date.html#1377">[ date ]</a>
<a href="index.html#1377">[ thread ]</a>
<a href="subject.html#1377">[ subject ]</a>
<a href="author.html#1377">[ author ]</a>
<a href="attachment.html">[ attachment ]</a>
</ul>
<hr>
<!-- body="start" -->
<p>
On Fri, 24 Jan 2003, Nathanael Allison wrote:
<br>
<p><em>&gt; Nate writes,
</em><br>
<p>(Oh joy Nate a substantive question that isn't about the stupid Iraq thing!)
<br>
<p>Thank you, thank you, let me wash your feet to thank you.
<br>
<p><em>&gt; I have read they will be done mapping out the human genome this year.
</em><br>
<p>Ok, lets discuss some terms.  &quot;Mapping&quot; in classical genetics is the
<br>
process of assigning some nanoscale sequence to some macroscale location.
<br>
<p>So for example, you can go to the &quot;Online Mendelian Inheritance in Man&quot;
<br>
database (otherwise known as OMIM), [<a href="http://www.ncbi.nlm.nih.gov/omim/">http://www.ncbi.nlm.nih.gov/omim/</a>]
<br>
<p>And if you go to the &quot;morbid map&quot; and type in &quot;Huntington Disease&quot;,
<br>
you will find that the gene for that disease maps to 4p16.3 (a specific
<br>
location on chromosone 4).
<br>
<p>This URL should pull it up:
<br>
&nbsp;&nbsp;<a href="http://www.ncbi.nlm.nih.gov/htbin-post/Omim/getmorbid?start=0&amp;term=Huntington+Disease">http://www.ncbi.nlm.nih.gov/htbin-post/Omim/getmorbid?start=0&amp;term=Huntington+Disease</a><br>
<p>It is worth noting that for every specific human disease which is believed
<br>
to have a &quot;genetic&quot; cause, it is assigned an OMIM entry and as information
<br>
about the disease accumulates, it becomes robustly documented.
<br>
<p>The OMIM number for Huntington Disease is 143100 and the online record
<br>
is here:
<br>
&nbsp;&nbsp;&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/dispomim.cgi?id=143100">http://www.ncbi.nlm.nih.gov/entrez/dispomim.cgi?id=143100</a>
<br>
<p>I'll simply note that the online description of Huntington's Disease
<br>
is in the vicinity of half a megabyte (larger than some novels) and
<br>
the last time I seriously worked with OMIM (close to a decade ago)
<br>
there were about 8000 human genetic diseases cataloged (many fewer
<br>
of them were mapped then than are now).
<br>
<p>So, the term &quot;mapping&quot; as dished out in the popular press is really
<br>
inaccurate (and I don't have a clue as to why it got promoted).
<br>
<p>The human genome (in terms of knowing the layout of the chromosomes,
<br>
where some of the disease genes resided, etc.) was largely &quot;mapped&quot;
<br>
by the mid-1990's if not earlier.
<br>
<p>What the press calls &quot;mapping&quot;, people who understand this call
<br>
&quot;sequencing&quot; -- i.e. one has to read the precise genetic sequence
<br>
of the A's, C's, G's, and T's that make up the DNA of the genetic
<br>
blueprint of an organism.  (So if one wants to be viewed as &quot;informed&quot;
<br>
one should lose the term &quot;mapping&quot;.)
<br>
<p>Now, the complete sequences for several of the smaller human chromosomes
<br>
(some of from 21-23 I think [higher numbered chromosomes are generally
<br>
much smaller and therefore easier to sequence]) were published several
<br>
years ago.  So they can be viewed as a &quot;done deal&quot;.  A number of genome
<br>
centers are working through the remaining larger chromosomes to resolve
<br>
the &quot;difficult&quot; to sequence parts.  (There are a number of reasons one
<br>
can encounter &quot;difficult&quot; parts -- but suffice to say they are only a
<br>
small fraction (perhaps 1-10%) of the overall sequence.)
<br>
<p>By and large the sequence (and mapping) of the human genome is *done*.
<br>
<p>That doesn't mean the locations (and genes responsible for) the 8000+
<br>
diseases in OMIM have been identified.  A decade or so ago, I think
<br>
only a few hundred disease genes &amp; locations were &quot;mapped&quot;.  Having the
<br>
complete sequence and a much more robust map is greatly accelerating
<br>
work in this area because it is much easier now to determine when
<br>
a sequence is &quot;abnormal&quot; and may therefore be the cause of a disease.
<br>
Identifying a mapped location, and then a &quot;gene&quot; responsible for a disease
<br>
greatly helps in determining what the function of an &quot;unknown gene&quot; is.
<br>
(This is a critical problem -- we do not know the functional purpose
<br>
of something like 30-50% of the genes we identify).  Though it is a
<br>
rather macabre situation, identifying the purpose of genes through
<br>
the diseases they cause is greatly accelerating medical progress.
<br>
<p><em>&gt; How long will it take to find out all the different combinations of each gene
</em><br>
<em>&gt; once they finish?
</em><br>
<p>This question isn't clear.  Gene variants are known as &quot;polymorphisms&quot; (&quot;many forms&quot;).
<br>
There is one gene that has multiple forms in the general human population -- ApoE which
<br>
has 3 variants (E2, E3 and E4) and those variants appear to play a critical role in ones
<br>
cholesterol levels and how long one lives.  There appear to be a large number of polymorphisms
<br>
in neurotransmitter receptors and transporters -- so many that I can't easily keep it straight --
<br>
this is Anders field)
<br>
(see: <a href="http://www.dna.com/capabilitiesServices/capabilitiesServices.jsp?site=dna&amp;link=cg_cnspanel.htm">http://www.dna.com/capabilitiesServices/capabilitiesServices.jsp?site=dna&amp;link=cg_cnspanel.htm</a><br>
<p>When a gene is severely broken (i.e. it has a polymorphism that produces an observable
<br>
disease) it will show up in OMIM and will probably be subject to analysis within the
<br>
next decade (or perhaps the next two decades).  When a gene is less severely broken
<br>
(people have a slightly lower sex drive or are somewhat less intelligent) it will
<br>
likely take much more time to determine whether there are genetic sources for these
<br>
conditions.
<br>
<p>The question of &quot;different combinations of each gene&quot; isn't clear to me.
<br>
Humans probably have 30-40,000 genes and perhaps some more RNA which may or
<br>
may not be exerting some significant influence.  With that number of factors,
<br>
I don't think one could try out all the different &quot;combinations&quot; with a
<br>
population *much* larger than exists in the world today.  So if the question
<br>
is about trying out all the different combinations -- it ain't going to happen.
<br>
<p><em>&gt; and what each gene changes in comparison to it's counterparts?
</em><br>
<p>For the reasons I cite above -- it isn't going to happen.  The best we
<br>
are going to manage in the near future is to determine the effects of
<br>
a few key gene variants in a few critical situations (e.g. the effect
<br>
of Apo E2/E3/E4 on cholesterol and longevity).
<br>
<p><em>&gt; Also what about genes present in some people and not in others?
</em><br>
<p>That is easy -- the URL I cite above (dna.com) seems to suggest that
<br>
we already have companies promoting services so you can perform
<br>
self-genetic diagnosis (otherwise known as &quot;genotyping&quot;).
<br>
<p><em>&gt; I would like to know a good estimate, if the technology is currently
</em><br>
<em>&gt; available or when the technology to make these discoveries will probably be
</em><br>
<em>&gt; available, thanks.
</em><br>
<p>The technology is available *now* that you could sequence parts of your genome
<br>
(or even your entire genome) if you have the resources to do so.  The URLs
<br>
I cite show that it looks like you can get the information out of ones genome
<br>
if one has the financial resources.  It will be moving in the direction
<br>
from &quot;I have an IQ or SAT score of X&quot; to &quot;I have a dopamine DRD3 receptor of Y&quot;.
<br>
<p>What isn't available &quot;now&quot; is the information that would determine (with a
<br>
high probability) that with a dopamine DRD3 receptor Y that you are the next
<br>
Steve Jobs or a sex crazed psycopath.  (In contrast the APO E2/E3/E4 data
<br>
is probably solid enough that one could set life insurance rates based on it.)
<br>
<p>If you have managed to get through this to this point you know more about
<br>
genomics than almost any college professor excepting those who can claim
<br>
&quot;ownership&quot; to a specific human chromosome (or those that have sequenced
<br>
the entire genome of some lesser organism).
<br>
<p>So go out and kick some butt.
<br>
R.
<br>
<!-- body="end" -->
<hr>
<ul>
<!-- next="start" -->
<li><strong>Next message:</strong> <a href="1378.html">Lee Corbin: "RE: Euro Trash Talk,"</a>
<li><strong>Previous message:</strong> <a href="1376.html">spike66: "sports"</a>
<li><strong>In reply to:</strong> <a href="1373.html">Nathanael Allison: "Questions about human genome project"</a>
<!-- nextthread="start" -->
<!-- reply="end" -->
<li><strong>Messages sorted by:</strong> 
<a href="date.html#1377">[ date ]</a>
<a href="index.html#1377">[ thread ]</a>
<a href="subject.html#1377">[ subject ]</a>
<a href="author.html#1377">[ author ]</a>
<a href="attachment.html">[ attachment ]</a>
</ul>
<!-- trailer="footer" -->
<hr>
<p><small><em>
This archive was generated by <a href="http://www.hypermail.org/">hypermail 2.1.5</a> 
: Sun Feb 02 2003 - 21:26:03 MST
</em></small></p>
</body>
</html>
