<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN"
                      "http://www.w3.org/TR/REC-html40/loose.dtd">
<HTML>
<HEAD>
<TITLE>extropians: Cellular aging [was Re: &gt;H A simulated cell]</TITLE>
<META NAME="Author" CONTENT="Robert Bradbury (bradbury@genebee.msu.su)">
<META NAME="Subject" CONTENT="Cellular aging [was Re: &gt;H A simulated cell]">
</HEAD>
<BODY BGCOLOR="#FFFFFF" TEXT="#000000">
<H1>Cellular aging [was Re: &gt;H A simulated cell]</H1>
<!-- received="Tue Jan 25 23:08:24 2000" -->
<!-- isoreceived="20000126060824" -->
<!-- sent="Wed, 26 Jan 2000 09:09:14 -0500 (EST)" -->
<!-- isosent="20000126140914" -->
<!-- name="Robert Bradbury" -->
<!-- email="bradbury@genebee.msu.su" -->
<!-- subject="Cellular aging [was Re: &gt;H A simulated cell]" -->
<!-- id="Pine.UW2.4.20.0001260809410.15326-100000@ilr" -->
<!-- inreplyto="014901bf679e$b83d9300$0487ecd1@neptune" -->
<STRONG>From:</STRONG> Robert Bradbury (<A HREF="mailto:bradbury@genebee.msu.su?Subject=Re:%20Cellular%20aging%20[was%20Re:%20&gt;H%20A%20simulated%20cell]&In-Reply-To=&lt;Pine.UW2.4.20.0001260809410.15326-100000@ilr&gt;"><EM>bradbury@genebee.msu.su</EM></A>)<BR>
<STRONG>Date:</STRONG> Wed Jan 26 2000 - 07:09:14 MST
<P>
<!-- next="start" -->
<UL>
<LI><STRONG>Next message:</STRONG> <A HREF="1561.html">Spudboy100@aol.com: "Re: What the future holds/Reality check"</A>
<LI><STRONG>Previous message:</STRONG> <A HREF="1559.html">Michael M. Butler: "Naeher mein Leitl zu Ihnen (was Re: (de Garis') are fake neurons)"</A>
<LI><STRONG>In reply to:</STRONG> <A HREF="1523.html">Technotranscendence: "Re: &gt;H A simulated cell"</A>
<!-- nextthread="start" -->
<LI><STRONG>Next in thread:</STRONG> <A HREF="1567.html">Technotranscendence: "Re: Cellular aging"</A>
<LI><STRONG>Reply:</STRONG> <A HREF="1567.html">Technotranscendence: "Re: Cellular aging"</A>
<LI><STRONG>Reply:</STRONG> <A HREF="1598.html">Eugene Leitl: "Cellular aging [was Re: &gt;H A simulated cell]"</A>
<!-- reply="end" -->
<LI><STRONG>Messages sorted by:</STRONG> 
<A HREF="date.html#1560">[ date ]</A>
<A HREF="index.html#1560">[ thread ]</A>
<A HREF="subject.html#1560">[ subject ]</A>
<A HREF="author.html#1560">[ author ]</A>
</UL>
<HR NOSHADE><P>
<!-- body="start" -->
<P>
On Tue, 25 Jan 2000, Technotranscendence wrote:
<BR>
<P><EM>&gt; 
</EM><BR>
<EM>&gt; Also, cryobiology is not the only field I mentioned.  Antiaging research is
</EM><BR>
<EM>&gt; the other.  What of that?  If one can develop a good model of aging at the
</EM><BR>
<EM>&gt; cellular or molecular level, then finding new ways to combat it would seem
</EM><BR>
<EM>&gt; to me to be a lot easier.
</EM><BR>
<P>Aeiveos Sciences Group started on that with the differential gene
<BR>
expression studies (at the tissue level).  The problem is the technology
<BR>
isn't available yet to provide the information base at the cellular
<BR>
level.   You could just about do it if you could tease out single
<BR>
cells from frozen tissues, do quantitative PCR on the mRNA in the
<BR>
cells and then do differential display analysis (to see how gene
<BR>
expression levels differ in young, middle-aged and old animals).
<BR>
But you are talking difficult methods and very high costs here.
<BR>
<P>You really need the gene expression chips for the human genome *and*
<BR>
a lab that can use lasers to excise single cells.  Otherwise the picture
<BR>
you get is that of tissue aggregates that tell you little about
<BR>
&quot;cellular aging&quot;.  Then there is a separate problem of how you get people
<BR>
to sit still for a few hundred needle biopsies into various organs (doctors
<BR>
and hospital facilities to do these don't come cheap).  You could do the
<BR>
studies in mice, but it remains an open question how well understanding
<BR>
aging in short-lived animals will translate to understanding aging in
<BR>
long-lived animals.  My feeling is that you will get a partial picture.
<BR>
I'd guess in 3-5 years researchers should be able to begin working
<BR>
on this aggressively with it only costing half an arm.
<BR>
<P><EM>&gt; From the looks of it, it appears most research in this area is
</EM><BR>
<EM>&gt; identify a mechanism, find something that inhibits it, look for
</EM><BR>
<EM>&gt; similar things to that inhibitor, and so forth.  Am I right?
</EM><BR>
<P>Yep.  Fundamentally however you have to deal with the problem of
<BR>
mutating DNA.  A point I made perhaps 7+ years ago on one of the
<BR>
aging newsgroups was as follows -- if ~30% of people die from cancer
<BR>
and it takes only ~5 mutations in specific sensitive locations in
<BR>
a subset of a few hundred tumor suppressor/promoter genes (those
<BR>
regulating cell division) in a *single* cell in your body, then *what*
<BR>
do you think is happening in all of the other genes in all of the cells
<BR>
that *aren't* involved in getting cancer?  [There is a really good
<BR>
paper in this idea that I should write now that Nanomedicine is out
<BR>
and most of the numbers are at my fingertips...]
<BR>
<P>There aren't going to be any *easy* mechanisms for inhibiting
<BR>
DNA damage that don't involve some major genetic enhancements.
<BR>
The best we might hope to obtain without some major re-engineering
<BR>
is flipping the switches to reduce-the-damage/increase-the-repair
<BR>
to levels found in children.  That might slow the accumulation, but I
<BR>
doubt it will stop it.
<BR>
<P><EM>&gt; 
</EM><BR>
<EM>&gt; A simulated aging cell could be used to more efficiently search for aging
</EM><BR>
<EM>&gt; inhibitors and therapies, no?
</EM><BR>
<EM>&gt; 
</EM><BR>
<P>Perhaps, but its way beyond our capabilities now.
<BR>
<P>NM [pg 115] - neuron volume: 14,000 microns^3, NM: [pg 384], molecular
<BR>
volume of water (ignoring solutes &amp; proteins since the density is not
<BR>
hugely different): 0.0299 nm^3/molecule.  If I'm doing the math right,
<BR>
that looks to be about 4.7x10^14 atoms.  Even a bacteria works out to
<BR>
1.5x10^11 atoms.  Given the max atomic level simulations now are at
<BR>
a billion atoms, if we assume a generous Moore's Law that hits no
<BR>
bumps providing a doubling every year, that means you don't get capacity
<BR>
for bacteria simulations until 2007 and neuron cell simulations until
<BR>
around 2018. The &quot;Blue Gene&quot; machine from IBM might speed this up
<BR>
a little but we are still up against the wall of computational limits.
<BR>
<P>However, the nice thing about this, is that since the code exists for
<BR>
modeling very precise radiation hazards and one would expect the
<BR>
free radical effects to be accounted for in the cell model, we will
<BR>
be able to model very accurately how and where the DNA is being damaged
<BR>
*and* test various interventions.
<BR>
<P>Then finally, we just might settle the entire damn anti-oxidant supplement
<BR>
debate once and for all.
<BR>
<P>Robert
<BR>
<P><!-- body="end" -->
<HR NOSHADE>
<UL>
<!-- next="start" -->
<LI><STRONG>Next message:</STRONG> <A HREF="1561.html">Spudboy100@aol.com: "Re: What the future holds/Reality check"</A>
<LI><STRONG>Previous message:</STRONG> <A HREF="1559.html">Michael M. Butler: "Naeher mein Leitl zu Ihnen (was Re: (de Garis') are fake neurons)"</A>
<LI><STRONG>In reply to:</STRONG> <A HREF="1523.html">Technotranscendence: "Re: &gt;H A simulated cell"</A>
<!-- nextthread="start" -->
<LI><STRONG>Next in thread:</STRONG> <A HREF="1567.html">Technotranscendence: "Re: Cellular aging"</A>
<LI><STRONG>Reply:</STRONG> <A HREF="1567.html">Technotranscendence: "Re: Cellular aging"</A>
<LI><STRONG>Reply:</STRONG> <A HREF="1598.html">Eugene Leitl: "Cellular aging [was Re: &gt;H A simulated cell]"</A>
<!-- reply="end" -->
<LI><STRONG>Messages sorted by:</STRONG> 
<A HREF="date.html#1560">[ date ]</A>
<A HREF="index.html#1560">[ thread ]</A>
<A HREF="subject.html#1560">[ subject ]</A>
<A HREF="author.html#1560">[ author ]</A>
</UL>
<!-- trailer="footer" -->
<HR NOSHADE>
<P>
<SMALL>
<EM>
This archive was generated by <A HREF="http://www.hypermail.org/">hypermail 2b29</A> 
: <EM>Thu Jul 27 2000 - 14:02:43 MDT</EM>
</EM>
</SMALL>
</BODY>
</HTML>
