<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN"
                      "http://www.w3.org/TR/html4/strict.dtd">
<html lang="en">
<head>
<meta name="generator" content="hypermail 2.1.5, see http://www.hypermail.org/">
<title>Extropians: Re: Removing lysosomal aggregates; obviating mitochondrial mutations (was: specific amino acid...)</title>
<meta name="Author" content="Aubrey de Grey (ag24@gen.cam.ac.uk)">
<meta name="Subject" content="Re: Removing lysosomal aggregates; obviating mitochondrial mutations (was: specific amino acid...)">
<meta name="Date" content="2003-04-23">
<style type="text/css">
body {color: black; background: #ffffff}
h1.center {text-align: center}
div.center {text-align: center}
</style>
</head>
<body>
<h1>Re: Removing lysosomal aggregates; obviating mitochondrial mutations (was: specific amino acid...)</h1>
<!-- received="Wed Apr 23 10:53:39 2003" -->
<!-- isoreceived="20030423165339" -->
<!-- sent="Wed, 23 Apr 2003 17:53:31 +0100" -->
<!-- isosent="20030423165331" -->
<!-- name="Aubrey de Grey" -->
<!-- email="ag24@gen.cam.ac.uk" -->
<!-- subject="Re: Removing lysosomal aggregates; obviating mitochondrial mutations (was: specific amino acid...)" -->
<!-- id="E198NUx-00000G-00@ag24" -->
<!-- inreplyto="Removing lysosomal aggregates; obviating mitochondrial mutations (was: specific amino acid...)" -->
<!-- expires="-1" -->
<p>
<strong>From:</strong> Aubrey de Grey (<a href="mailto:ag24@gen.cam.ac.uk?Subject=Re:%20Removing%20lysosomal%20aggregates;%20obviating%20mitochondrial%20mutations%20(was:%20specific%20amino%20acid...)"><em>ag24@gen.cam.ac.uk</em></a>)<br>
<strong>Date:</strong> Wed Apr 23 2003 - 10:53:31 MDT
</p>
<!-- next="start" -->
<li><strong>Next message:</strong> <a href="7092.html">Robert J. Bradbury: "Re: Genetic engineering costs"</a>
<ul>
<li><strong>Previous message:</strong> <a href="7090.html">gts: "RE: PHIL: Good question"</a>
<li><strong>Maybe in reply to:</strong> <a href="7009.html">Aubrey de Grey: "Removing lysosomal aggregates; obviating mitochondrial mutations (was: specific amino acid...)"</a>
<!-- nextthread="start" -->
<li><strong>Next in thread:</strong> <a href="7194.html">Rafal Smigrodzki: "RE: Removing lysosomal aggregates; obviating mitochondrial mutations (was: specific amino acid...)"</a>
<li><strong>Reply:</strong> <a href="7194.html">Rafal Smigrodzki: "RE: Removing lysosomal aggregates; obviating mitochondrial mutations (was: specific amino acid...)"</a>
<!-- reply="end" -->
<li><strong>Messages sorted by:</strong> 
<a href="date.html#7091">[ date ]</a>
<a href="index.html#7091">[ thread ]</a>
<a href="subject.html#7091">[ subject ]</a>
<a href="author.html#7091">[ author ]</a>
<a href="attachment.html">[ attachment ]</a>
</ul>
<hr>
<!-- body="start" -->
<p>
Rafal Smigrodzki wrote:
<br>
<p><em>&gt; Tarassov
</em><br>
<p>This is indeed cool, but not obviously going anywhere therapeutically.
<br>
No one (even evolution) has succeeded in getting big RNAs into the
<br>
mitochondrion, and for this to work we would need lots of them (the
<br>
two ribosomal RNAs and also at least 11 messenger RNAs).  For protein
<br>
import all we have to do is hijack a pathway that already works fine
<br>
for 1000 other proteins and use it for 13 more -- much easier.
<br>
<p><em>&gt; Guy
</em><br>
<p>Right.  This is the second one to be got working in mammalian cells;
<br>
ATP6 is the other (Nat Genet. 2002 Apr;30(4):394-9).
<br>
<p><em>&gt; only two mt proteins have *not* been found in the nucleus in one
</em><br>
<em>&gt; organism or the other.
</em><br>
<p>Wrong (though often stated).  Only two (cytochrome oxidase I and
<br>
cytochrome b) have always been found in the mt, but that's not the
<br>
same, because Complex I is completely absent in some yeasts and in
<br>
Plasmodium -- they get by with an enzyme that does the electron
<br>
transport but no proton pumping (for which they manage with the
<br>
contributions of Complexes III and IV).  There are four subunits of
<br>
Complex I for which these are the only species that lack the genes in
<br>
the mtDNA, and of course they also lack them in the nDNA.
<br>
<p><em>&gt; Possibly the reason for stability of the mtDNA in
</em><br>
<em>&gt; evolution is not the protein import problem, but the difference in the
</em><br>
<em>&gt; nuclear and mito genetic code.
</em><br>
<p>That works for animals, but not plants (see my mail of yesterday).
<br>
<p><em>&gt; ### Human mt-mRNA is polycistronic.
</em><br>
<p>Sort of..... it's transcribed as one message for each strand, but they
<br>
are then chopped up and each bit polyadenylated before translation.
<br>
Two of the protein-coding products of the chopping-up are dicistronic,
<br>
the rest mono.
<br>
<p><em>&gt; my article on mt mutations in PD is written up, we are just working on
</em><br>
<em>&gt; the text. I can't tell you the details but let me say one thing - there
</em><br>
<em>&gt; is a lot more of them than most people thought.
</em><br>
<p>I know of some other work in this area (but likewise I'm afraid I can't
<br>
divulge much at this point).  I can say, though, that the question of
<br>
what causes what is made even more fraught by this data.
<br>
<p><em>&gt; If there is a different underlying process, dealing with the aggregates
</em><br>
<em>&gt; will not remove all of the problem, although it could ameliorate some of it.
</em><br>
<em>&gt; I would rather try to attack the ultimate cause, which most likely doesn't
</em><br>
<em>&gt; involve the proteins that actually aggregate
</em><br>
<p>That depends.  If there is a different underlying process that does not
<br>
exert its pathogenicity via the aggregates, then I agree, but that's a
<br>
big &quot;if&quot;.  If it does, then getting rid of the aggregates is a complete
<br>
solution -- and an ideal one, because we didn't have to mess about with
<br>
normal metabolic processes to try to make them &quot;cleaner&quot;.
<br>
<p><em>&gt; in the absence of Elan, amyloid is extralysosomal, so it
</em><br>
<em>&gt; couldn't act by choking the lysosomes, and putting xenoenzymes in this
</em><br>
<em>&gt; compartment will not allow amyloid removal.
</em><br>
<p>Right.  It's possible that engineering transgenic microbial hydrolases
<br>
to be secreted to the extracellular medium would work, but I think that
<br>
would be more likely to have side-effects.  I prefer the Elan approach
<br>
in conjunction with lysosomal enhancement.
<br>
<p><em>&gt; Is there some alpha-synuclein in lysosomes simply because stuff ends up
</em><br>
<em>&gt; in lysosomes, or is it's presence in the lysosomes the mechanism of its
</em><br>
<em>&gt; detrimental action?  This has not been answered so far, AFAIK.
</em><br>
<p>I agree, but who cares? -- the only thing that matters is whether NON-
<br>
lysosomal alpha-synuclein is bad for cells, and no one has postulated 
<br>
any way that it could be.
<br>
<p><em>&gt; ### Atheromatous material may be difficult to digest, but is not
</em><br>
<em>&gt; non-digestible. With some treatments you can have regression of
</em><br>
<em>&gt; atherosclerotic changes implying digestion of macroscopic amounts of
</em><br>
<em>&gt; plaque. There is a dynamic balance between plaque accumulation and
</em><br>
<em>&gt; digestion.
</em><br>
<p>That doesn't follow.  At any given instant, there will certainly be a
<br>
proportion of the atheroma core that's digestible (and is indeed being
<br>
digested).  That proportion will diminish as a result of treatments
<br>
that slow down the arrival of new material, etc., so the total size
<br>
of the plaque will also diminish.  But that doesn't tell us whether
<br>
there is *also* stuff there that *can't* be degraded.  In fact we can
<br>
say with confidence that there is, because the plaque appears out of
<br>
nothing.  The originating macrophages that enter our artery wall in
<br>
youth, for example, can only become foam cells by hanging out ther
<br>
and taking in a steady stream of LDL etc and failing to break it all
<br>
down -- else they would remain as healthy macrophages forever.  Same
<br>
goes for the core of the mature plaque -- there's a constant stream
<br>
on new macrophages arriving all the time to do battle with it, and
<br>
if they were able to break down everything there they would do so.
<br>
You're quite right that macrophages can eat incredibly tough stuff
<br>
-- good thing too, or we would have atherosclerosis a lot younger --
<br>
but oxidised cholesterol is *really* tough stuff.
<br>
<p><p>Robert Bradbury wrote:
<br>
<p><em>&gt; Has there been any work to determine whether the lower metabolic
</em><br>
<em>&gt; rate directly correlates with a lower cancer rate for their mass
</em><br>
<em>&gt; (or # of cells)?  I.e. do they have better cancer defenses than we
</em><br>
<em>&gt; do or is it indeed due precisely to lower free radical production?
</em><br>
<p>Do you mean, are elephant/whale cells more resistant to transformation
<br>
in vitro (where the O2 tension and consumption can be controlled)?  I
<br>
believe Jerry Shay is working on this, but it's a hard question because
<br>
you have to do such a lot to human cells to transform them that there's
<br>
no reason to suppose that an organism with equally good (but no better)
<br>
defences would have exactly the same defences.
<br>
<p><em>&gt; I was under the impression that
</em><br>
<em>&gt; there was a significant variance in mtDNA sizes
</em><br>
<p>Not in animals -- 15 to 18 kb.  Reviews of this sort of thing come out
<br>
regularly; also see &lt;<a href="http://megasun.bch.umontreal.ca/ogmpproj.html">http://megasun.bch.umontreal.ca/ogmpproj.html</a>&gt;.
<br>
<p><em>&gt; I also disagree that evolution could not pull a rabbit out of the hat.
</em><br>
<em>&gt; It would depend entirely on how many UGAs there would be in a specific
</em><br>
<em>&gt; mtGene.  I consider the prospect of adding a mt Signal Sequence to be
</em><br>
<em>&gt; much more difficult than fixing the UGA problem.  After all one cannot
</em><br>
<em>&gt; know how frequently mutated mtDNA might substitute all of the UGA codes
</em><br>
<em>&gt; with UGG codons (allowing an alternate Trp coding) and subsequently
</em><br>
<em>&gt; transfer the genes to the nucleus.
</em><br>
<p>It's certainly true that biologists are still very bad at guessing what
<br>
evolution will find easy and what hard.  But we have plenty of data on
<br>
this; gene transfer to the nucleus when there is no code disparity and
<br>
no hydrophobicity problem has happened hundreds of times in evolution
<br>
(so apparently gaining a signal sequence is easy), whereas there is not
<br>
a single recorded case of gene transfer occurring across code disparity
<br>
(i.e. of the most economical phylogenetic tree implying that).
<br>
<p><em>&gt; <a href="http://users.rcn.com/jkimball.ma.ultranet/BiologyPages/C/Codons.html">http://users.rcn.com/jkimball.ma.ultranet/BiologyPages/C/Codons.html</a>
</em><br>
<p>Indeed.  But the UGA switch probably matters most, because it causes a
<br>
subsequently transferred gene to encode a truncated protein.
<br>
<p><em>&gt; there isn't a lot of conclusive evidence
</em><br>
<em>&gt; that you can reduce the gene hydrophobicity enough to get all of the
</em><br>
<em>&gt; genes into the nucleus so one may always be left with some mtDNA.
</em><br>
<p>The evidence is now pretty compelling, actually.  We know that there
<br>
is a degree of hydrophobicity of each transmembrane segment that is
<br>
enough to make it stable in the membrane when it's assembled into a
<br>
complex but not enough to preclude import.  The mt-coded proteins are
<br>
thus &quot;unnecessarily&quot; hydrophobic and I am now very confident that we
<br>
can tweak them pretty quickly into an importable but still fuctional
<br>
form.  See, e.g., PNAS 2002 Aug 6;99(16):10510: just changing **two**
<br>
amino acids makes all the difference for one protein.
<br>
<p><em>&gt; Another question is whether the SO is being made by &quot;perfect&quot;
</em><br>
<em>&gt; proteins or damaged proteins?  If the proteins are becoming
</em><br>
<em>&gt; damaged and increasing SO production then we are back to the
</em><br>
<em>&gt; protein recycling issue all over again.
</em><br>
<p>We're back to the protein recycling issue either way.  In my book I
<br>
suggested that complete allotopic expression might double lifespan,
<br>
but I definitely only said there was a 10% chance of this... these
<br>
days I'm more interested in a more comprehensive onslaught on aging,
<br>
and protein recycling (i.e. lipofuscin elimination) is a key plank
<br>
of my approach.
<br>
<p><p>Rafal Smigrodzki wrote:
<br>
<p><em>&gt; it is indeed possible to replace full length mtDNA now, so it's
</em><br>
<em>&gt; just a matter of time before somebody manages to do something useful
</em><br>
<em>&gt; with it.
</em><br>
<p>Probably not in respect of aging.  Unfortunately it seems that the
<br>
mutant mtDNA molecule has a selective advantage in post-mitotic cells
<br>
and clonally expands at the expense of the wild-type, eventually
<br>
taking over the whole cell.  This process would overcome any attempt
<br>
to replace mutant mtDNA by wild-type copies.  No one has come up with
<br>
any feasible way to reverse this selective pressure (and not for want
<br>
of trying!).  Putting in mtDNA that reduces superoxide production
<br>
may be a different matter though -- but (broken record time) that's
<br>
just retardation, not repair/reversal/obviation.
<br>
<p><p>Damien Broderick wrote:
<br>
<p><em>&gt; I don't recall whether you've posted here until recently
</em><br>
<p>No, I just signed on a few days ago.  I'm already glad I chose the
<br>
digest version...
<br>
<p>Aubrey de Grey
<br>
<!-- body="end" -->
<hr>
<ul>
<!-- next="start" -->
<li><strong>Next message:</strong> <a href="7092.html">Robert J. Bradbury: "Re: Genetic engineering costs"</a>
<li><strong>Previous message:</strong> <a href="7090.html">gts: "RE: PHIL: Good question"</a>
<li><strong>Maybe in reply to:</strong> <a href="7009.html">Aubrey de Grey: "Removing lysosomal aggregates; obviating mitochondrial mutations (was: specific amino acid...)"</a>
<!-- nextthread="start" -->
<li><strong>Next in thread:</strong> <a href="7194.html">Rafal Smigrodzki: "RE: Removing lysosomal aggregates; obviating mitochondrial mutations (was: specific amino acid...)"</a>
<li><strong>Reply:</strong> <a href="7194.html">Rafal Smigrodzki: "RE: Removing lysosomal aggregates; obviating mitochondrial mutations (was: specific amino acid...)"</a>
<!-- reply="end" -->
<li><strong>Messages sorted by:</strong> 
<a href="date.html#7091">[ date ]</a>
<a href="index.html#7091">[ thread ]</a>
<a href="subject.html#7091">[ subject ]</a>
<a href="author.html#7091">[ author ]</a>
<a href="attachment.html">[ attachment ]</a>
</ul>
<!-- trailer="footer" -->
<hr>
<p><small><em>
This archive was generated by <a href="http://www.hypermail.org/">hypermail 2.1.5</a> 
: Wed Apr 23 2003 - 11:04:36 MDT
</em></small></p>
</body>
</html>
