<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN"
                      "http://www.w3.org/TR/REC-html40/loose.dtd">
<HTML>
<HEAD>
<TITLE>extropians: Re: Neuron Computational Requirements?</TITLE>
<META NAME="Author" CONTENT="Anders Sandberg (asa@nada.kth.se)">
<META NAME="Subject" CONTENT="Re: Neuron Computational Requirements?">
</HEAD>
<BODY BGCOLOR="#FFFFFF" TEXT="#000000">
<H1>Re: Neuron Computational Requirements?</H1>
<!-- received="Sat Apr 22 05:30:37 2000" -->
<!-- isoreceived="20000422113037" -->
<!-- sent="22 Apr 2000 13:30:49 +0200" -->
<!-- isosent="20000422113049" -->
<!-- name="Anders Sandberg" -->
<!-- email="asa@nada.kth.se" -->
<!-- subject="Re: Neuron Computational Requirements?" -->
<!-- id="b49em7yid92.fsf@sans04.nada.kth.se" -->
<!-- inreplyto="Thu, 20 Apr 2000 09:40:47 -0400&quot;" -->
<STRONG>From:</STRONG> Anders Sandberg (<A HREF="mailto:asa@nada.kth.se?Subject=Re:%20Neuron%20Computational%20Requirements?&In-Reply-To=&lt;b49em7yid92.fsf@sans04.nada.kth.se&gt;"><EM>asa@nada.kth.se</EM></A>)<BR>
<STRONG>Date:</STRONG> Sat Apr 22 2000 - 05:30:49 MDT
<P>
<!-- next="start" -->
<UL>
<LI><STRONG>Next message:</STRONG> <A HREF="1361.html">Anders Sandberg: "Re: NEWS: Rewiring the Brain"</A>
<LI><STRONG>Previous message:</STRONG> <A HREF="1359.html">Anders Sandberg: "Re: Posthuman Easter greets"</A>
<!-- nextthread="start" -->
<LI><STRONG>Next in thread:</STRONG> <A HREF="1483.html">Robin Hanson: "Re: Neuron Computational Requirements?"</A>
<LI><STRONG>Reply:</STRONG> <A HREF="1483.html">Robin Hanson: "Re: Neuron Computational Requirements?"</A>
<!-- reply="end" -->
<LI><STRONG>Messages sorted by:</STRONG> 
<A HREF="date.html#1360">[ date ]</A>
<A HREF="index.html#1360">[ thread ]</A>
<A HREF="subject.html#1360">[ subject ]</A>
<A HREF="author.html#1360">[ author ]</A>
</UL>
<HR NOSHADE><P>
<!-- body="start" -->
<P>
Robin Hanson &lt;<A HREF="mailto:rhanson@gmu.edu?Subject=Re:%20Neuron%20Computational%20Requirements?&In-Reply-To=&lt;b49em7yid92.fsf@sans04.nada.kth.se&gt;">rhanson@gmu.edu</A>&gt; writes:
<BR>
<P><EM>&gt; To me the most interesting questions are how good uploading technology
</EM><BR>
<EM>&gt; is now, how fast it is improving, and when therefore uploading may
</EM><BR>
<EM>&gt; be feasible.  The whole point of the uploading strategy to achieving
</EM><BR>
<EM>&gt; artificial intelligence is to port the software of a human brain
</EM><BR>
<EM>&gt; *without* understanding how it works.  The simplest strategy of this
</EM><BR>
<EM>&gt; sort is to just understand how each type of neuron works, and scan a
</EM><BR>
<EM>&gt; brain noting which neurons are of which types and are connected to
</EM><BR>
<EM>&gt; which other neurons.
</EM><BR>
<EM>&gt; 
</EM><BR>
<EM>&gt; Focusing on this neuron-level strategy, I want estimates for five
</EM><BR>
<EM>&gt; parameters.  Two are static parameters:
</EM><BR>
<EM>&gt; 
</EM><BR>
<EM>&gt; 1) Neuron Inventory -- The number of distinct types of neurons, and
</EM><BR>
<EM>&gt; the number of each type in a typical human brain.
</EM><BR>
<P>I don't have my textbooks at hand, but we are still a way off a
<BR>
complete list. The problem is that it is not obvious when to
<BR>
categorize neurons into distinct types or not
<BR>
(<A HREF="http://mickey.utmem.edu/papers/NUMBER_REV_1988.html#3">http://mickey.utmem.edu/papers/NUMBER_REV_1988.html#3</A> has a good
<BR>
explanation) - some vary continously along various dimensions, and
<BR>
subtle biochemical differences might be hard to detect. Still, if the
<BR>
grasshopper represents the largest possible number of unique neurons
<BR>
(~200,000) we have an upper bound which is almost certainly much
<BR>
larger than the number found in the human brain. 
<BR>
<P>I think a likely number of functionally different types is perhaps
<BR>
~1000. I'll check with my copy of Shepherd, _The Synaptic Organization
<BR>
of the Brain_ (it usually has good numbers). 
<BR>
<P>[ Robert mentioned that maybe 25% of the genome is involved in the brain
<BR>
development.  I have seen numbers of 40,000 genes expressed in brain
<BR>
development, based on
<BR>
<P>@Article{Adams93,
<BR>
&nbsp;&nbsp;author = 	 {MD. Adams and MB Soares and AR Kerlavage and C Fields and JC Venter},
<BR>
&nbsp;&nbsp;title = 	 {Rapid c{DNA} sequencing (expressed sequence tags) from a directionally cloned human infant brain cDNA library},
<BR>
&nbsp;&nbsp;journal = 	 {Nat Genet},
<BR>
&nbsp;&nbsp;year = 	 1993,
<BR>
&nbsp;&nbsp;volume =	 4,
<BR>
&nbsp;&nbsp;pages =	 {373--380}
<BR>
}
<BR>
<P>@Article{Sutcliffe88,
<BR>
&nbsp;&nbsp;author = 	 {J.G. Sutcliffe},
<BR>
&nbsp;&nbsp;title = 	 { m{RNA} in the mammalian central nervous system},
<BR>
&nbsp;&nbsp;journal = 	 {Annu Rev Neurosci},
<BR>
&nbsp;&nbsp;year = 	 1988,
<BR>
&nbsp;&nbsp;volume =	 11,
<BR>
&nbsp;&nbsp;pages =	 {157--198}
<BR>
} 
<BR>
]
<BR>
<P>40,000 genes is enough to create a near infinite number of unique
<BR>
neurons using self-organisation and morphogen gradients, but it is
<BR>
unlikely that evolution has favored that in mammals; instead many of
<BR>
these genes (beside the metabolism stuff) likely control gross
<BR>
connectivity - which neuron type to connect to which other where and
<BR>
how.
<BR>
<P><EM>&gt; 3) Resolution required -- The spatial/chemical resolution of a scanning
</EM><BR>
<EM>&gt; technology that would be required to sufficiently reliably distinguish
</EM><BR>
<EM>&gt; between the types of neurons, and to distinguish which neurons connect
</EM><BR>
<EM>&gt; to which others via which kinds of synapses.  (Assume one is repeated
</EM><BR>
<EM>&gt; slicing and scanning a cryogenically frozen brain.)
</EM><BR>
<P>We need to identify synapses, which means on the order of ~1 mu. At
<BR>
this resolution the morphological differences can probably be totally
<BR>
characterized, but we need the ability to get the receptor types on
<BR>
surfaces and possibly some chemical stuff from the inside too. I don't
<BR>
think we need much smaller resolution, since then we get down into the
<BR>
region where noise and diffusion play a significant role.
<BR>
<P><P><EM>&gt; Three parameters are dynamic, about *rates* of progress of these:
</EM><BR>
<EM>&gt; 
</EM><BR>
<EM>&gt; 1) Neuron types modeled -- The fraction of neuron types for which
</EM><BR>
<EM>&gt; we have reasonable computational models at *any* level of complexity.
</EM><BR>
<EM>&gt; That is, where we have run such models and seen that they correspond
</EM><BR>
<EM>&gt; reasonably closely to the behavior of actual neurons of that type.
</EM><BR>
<P>I think this can be estimated using a bibliometric study.
<BR>
<P><A HREF="http://theta.hippo.hscbklyn.edu/fox/Neuron.User.Manual/intro.html">http://theta.hippo.hscbklyn.edu/fox/Neuron.User.Manual/intro.html</A> has
<BR>
an interesting diagram (power.gif) which I think has held true since
<BR>
it was made. Knowledge of channels has grown enormously, but I think
<BR>
we will see a slowing down over time as the number of channels are
<BR>
finite. Computer hardware has grown faster than the abilities of
<BR>
simulation software, which has grown much faster than knowledge of the
<BR>
detailled morphology.
<BR>
<P><EM>&gt; 2) Average CPU/RAM per neuron -- A weighted average across neuron
</EM><BR>
<EM>&gt; types of the CPU/RAM requirements for simulating each neuron type.
</EM><BR>
<EM>&gt; The weights should be according to the fraction of each type of
</EM><BR>
<EM>&gt; neuron in a human brain.
</EM><BR>
<P>I think this can be estimated roughly for the detailled models (like
<BR>
the Purkinje model or Traub's pyramidal model), although I don't have
<BR>
the numbers around. There is likely a multiplicative states x
<BR>
compartment demand on memory, where models with many compartments will
<BR>
produce heavier load than morphologically simple neurons. Those
<BR>
Purkinje cells (one of the most branching types around) will likely be
<BR>
heavy, while we can hope the granule cells are lighter. CPU estimates
<BR>
are harder, as there are a lot of clever simplifications and tricks
<BR>
that can be done (one example is Alain Destexhe's trick of using
<BR>
&quot;canned&quot; action potentials instead of exactly calculated (very heavy)
<BR>
ones and instead concentrate on the dynamics when the neuron is not
<BR>
firing).
<BR>
&nbsp;
<BR>
<EM>&gt; 3) Scanning costs -- The time/dollar cost per area scanned of scanning
</EM><BR>
<EM>&gt; at the resolution required for uploading.  Costs of scanning at
</EM><BR>
<EM>&gt; other resolutions may be interesting if time trends in them can be
</EM><BR>
<EM>&gt; more accurately estimates, and if one can estimate a scaling
</EM><BR>
<EM>&gt; relationship between the costs of scanning at different resolutions.
</EM><BR>
<P>I think we might see a nice decrease here. I have met Bruce
<BR>
H. McCormick from Texas A&amp;M University at the CNS conferences, and he
<BR>
is proposing an automated microtome linked to a CCD camera that would
<BR>
be able to slice and scan tissues at a very high rate at a quite
<BR>
modest price. Of course, his proposal would currently just help get
<BR>
the morphology of single dyed neurons, but after talking to him I got
<BR>
the impression that the scanning costs can be decreased quite a
<BR>
bit. It might not scale all the way down to 1 mu with an entire brain,
<BR>
but it looks positive.
<BR>
<P><EM>&gt; The progress estimate of neuron types modeled help us estimate
</EM><BR>
<EM>&gt; when it will be possible to upload at any cost, while the other
</EM><BR>
<EM>&gt; estimates (together with estimates of how fast CPU/RAM costs fall)
</EM><BR>
<EM>&gt; help us estimate how fast the cost of uploading is falling with time.
</EM><BR>
<EM>&gt; 
</EM><BR>
<EM>&gt; Anders, tell me if I'm wrong, but it strikes me that we may well have
</EM><BR>
<EM>&gt; enough experience modeling neurons to start to estimate these parameters.
</EM><BR>
<EM>&gt; If so, constructing such estimates of when uploading will be feasible
</EM><BR>
<EM>&gt; and how fast costs will fall seems a very high priority to me.
</EM><BR>
<P>I think we can do a reasonable estimate for this. In fact, since I am
<BR>
thinking of doing a talk at TransVision in London this summer on
<BR>
uploading, it seems to be a good idea to do. I'll see what numbers I
<BR>
can dig up.
<BR>
<P><PRE>
-- 
-----------------------------------------------------------------------
Anders Sandberg                                      Towards Ascension!
<A HREF="mailto:asa@nada.kth.se?Subject=Re:%20Neuron%20Computational%20Requirements?&In-Reply-To=&lt;b49em7yid92.fsf@sans04.nada.kth.se&gt;">asa@nada.kth.se</A>                            <A HREF="http://www.nada.kth.se/~asa/">http://www.nada.kth.se/~asa/</A>
GCS/M/S/O d++ -p+ c++++ !l u+ e++ m++ s+/+ n--- h+/* f+ g+ w++ t+ r+ !y
</PRE>
<P><!-- body="end" -->
<HR NOSHADE>
<UL>
<!-- next="start" -->
<LI><STRONG>Next message:</STRONG> <A HREF="1361.html">Anders Sandberg: "Re: NEWS: Rewiring the Brain"</A>
<LI><STRONG>Previous message:</STRONG> <A HREF="1359.html">Anders Sandberg: "Re: Posthuman Easter greets"</A>
<!-- nextthread="start" -->
<LI><STRONG>Next in thread:</STRONG> <A HREF="1483.html">Robin Hanson: "Re: Neuron Computational Requirements?"</A>
<LI><STRONG>Reply:</STRONG> <A HREF="1483.html">Robin Hanson: "Re: Neuron Computational Requirements?"</A>
<!-- reply="end" -->
<LI><STRONG>Messages sorted by:</STRONG> 
<A HREF="date.html#1360">[ date ]</A>
<A HREF="index.html#1360">[ thread ]</A>
<A HREF="subject.html#1360">[ subject ]</A>
<A HREF="author.html#1360">[ author ]</A>
</UL>
<!-- trailer="footer" -->
<HR NOSHADE>
<P>
<SMALL>
<EM>
This archive was generated by <A HREF="http://www.hypermail.org/">hypermail 2b29</A> 
: <EM>Thu Jul 27 2000 - 14:09:41 MDT</EM>
</EM>
</SMALL>
</BODY>
</HTML>
