<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN"
                      "http://www.w3.org/TR/REC-html40/loose.dtd">
<html>
<head>
<title>extropians: Re: How many genes do we differ on?</title>
<meta name="Author" content="hal@finney.org (hal@finney.org)">
<meta name="Subject" content="Re: How many genes do we differ on?">
</head>
<body bgcolor="#FFFFFF" text="#000000">
<h1>Re: How many genes do we differ on?</h1>
<!-- received="Fri Feb 16 20:04:59 2001" -->
<!-- isoreceived="20010217030459" -->
<!-- sent="Fri, 16 Feb 2001 19:02:48 -0800" -->
<!-- isosent="20010217030248" -->
<!-- name="hal@finney.org" -->
<!-- email="hal@finney.org" -->
<!-- subject="Re: How many genes do we differ on?" -->
<!-- id="200102170302.TAA15830@finney.org" -->
<!-- inreplyto="How many genes do we differ on?" -->
<strong>From:</strong> <a href="mailto:hal@finney.org?Subject=Re:%20How%20many%20genes%20do%20we%20differ%20on?&In-Reply-To=&lt;200102170302.TAA15830@finney.org&gt;"><em>hal@finney.org</em></a><br>
<strong>Date:</strong> Fri Feb 16 2001 - 20:02:48 MST
<p>
<!-- next="start" -->
<ul>
<li><strong>Next message:</strong> <a href="4046.html">hal@finney.org: "Re: Is the U.S socialist? (was Re: Neal Blaike vs Michael Lorrey)"</a>
<li><strong>Previous message:</strong> <a href="4044.html">Jerry Mitchell: "RE: Is the U.S socialist? (was Re: Neal Blaike vs Michael Lorrey)"</a>
<li><strong>Maybe in reply to:</strong> <a href="4015.html">Robin Hanson: "How many genes do we differ on?"</a>
<!-- nextthread="start" -->
<!-- reply="end" -->
<li><strong>Messages sorted by:</strong> 
<a href="date.html#4045">[ date ]</a>
<a href="index.html#4045">[ thread ]</a>
<a href="subject.html#4045">[ subject ]</a>
<a href="author.html#4045">[ author ]</a>
</ul>
<hr noshade><p>
<!-- body="start" -->
<p>
<a href="mailto:CurtAdams@aol.com?Subject=Re:%20How%20many%20genes%20do%20we%20differ%20on?&In-Reply-To=&lt;200102170302.TAA15830@finney.org&gt;">CurtAdams@aol.com</a> writes:
<br>
<em>&gt; I don't have exact numbers to hand, but Futuyama cites large mammals
</em><br>
<em>&gt; as being heterozygous (having different genes) at 3.7% of allozymes.  That
</em><br>
<em>&gt; means 3.7% of the time your two alleles produce proteins of significantly
</em><br>
<em>&gt; different motility.  Since most variants are rare, two people would differ
</em><br>
<em>&gt; at about 7% of loci; with 30,000 loci now estimated for the human population
</em><br>
<em>&gt; that's a floor of 2,100 genes.  It will be higher as most changes aren't
</em><br>
<em>&gt; electrophoretically detectable.
</em><br>
<p><em>&gt;From one of the Nature papers [1] I find:
</em><br>
<p>&nbsp;&nbsp;&nbsp;The allele frequencies of a set of SNPs have been evaluated in
<br>
&nbsp;&nbsp;&nbsp;independent populations using pooled resequencing. Samples of TSC (n =
<br>
&nbsp;&nbsp;&nbsp;502) and overlap SNPs (n = 774) were studied in population samples
<br>
&nbsp;&nbsp;&nbsp;of European, African American and Chinese descent, revealing 82%
<br>
&nbsp;&nbsp;&nbsp;to be polymorphic in at least one ethnic group at frequencies above
<br>
&nbsp;&nbsp;&nbsp;the detection threshold of pooled resequencing (10%). The remaining
<br>
&nbsp;&nbsp;&nbsp;18% presumably represent SNPs with a frequency less than 10% in
<br>
&nbsp;&nbsp;&nbsp;the populations surveyed and false positives. Furthermore, 77% of
<br>
&nbsp;&nbsp;&nbsp;SNPs had a minor allele frequency of more than 20% in at least one
<br>
&nbsp;&nbsp;&nbsp;population, and 27% had an allele frequency higher than 20% in all
<br>
&nbsp;&nbsp;&nbsp;three ethnic groups. TSC and overlap SNPs had similar distributions
<br>
&nbsp;&nbsp;&nbsp;across the populations, showing that they are comparable in quality
<br>
&nbsp;&nbsp;&nbsp;and frequency. The high proportion of SNPs with significant population
<br>
&nbsp;&nbsp;&nbsp;frequency is expected after SNP discovery in two or a few chromosomes,
<br>
&nbsp;&nbsp;&nbsp;given standard assumptions about human population history.
<br>
<p>As I read this, 77% of SNPs had the less popular variation in at least 20%
<br>
of the individuals in at least one of the three ethnic groups, European,
<br>
African-American, and Chinese.  This implies that in the group that
<br>
experience the most variation at this SNP, the chance of two randomly
<br>
chosen individuals varying is at least 32% (2*.8*.2).  27% of SNPs had
<br>
this level of variation in all three groups.
<br>
<p>I can't quite work out the math from all this, but broadly speaking
<br>
it looks to me like 10-20% of SNPs are likely to actually vary in two
<br>
randomly chosen individuals from the population of the world.
<br>
<p>The paper further estimates about 2 SNPs per gene, for a total of about
<br>
60000 SNPs in expressed DNA.  Combining this with my previous estimate
<br>
we get on the order of 10000 genetic differences in two randomly chosen
<br>
individuals.
<br>
<p>Another article [2] puts it this way:
<br>
<p>&nbsp;&nbsp;&nbsp;However, the real importance of SNPs is that there are so many of
<br>
&nbsp;&nbsp;&nbsp;them. One estimate is that comparing two human DNA sequences results
<br>
&nbsp;&nbsp;&nbsp;in a SNP every 1,000-2,000 nucleotides. That may not sound like much
<br>
&nbsp;&nbsp;&nbsp;until you realize that there are 3.2 billion nucleotides in the human
<br>
&nbsp;&nbsp;&nbsp;genome, which translates into 1.6 million-3.2 million SNPs. And that's
<br>
&nbsp;&nbsp;&nbsp;just from comparing two sequences - the total number of SNPs in humans
<br>
&nbsp;&nbsp;&nbsp;is obviously much more.
<br>
<p>So here we have the 3 million SNP figure again, this time in the context
<br>
of two (presumably) randomly chosen individuals.  However most of these
<br>
will be in non-coding regions.  I can't find a clear statement of what
<br>
percentage of the genome represents coding regions, but I find &quot;a few
<br>
percent&quot; mentioned here and there.  That would be about 100,000 SNPs
<br>
between two individuals.
<br>
<p>These two figures are not very consistent, and neither corresponds too
<br>
well with Curt's estimate of 2100.  It's possible that the second paper
<br>
was wrong to say that the 3 million SNP value was from two individuals,
<br>
but rather it may have been the collective data from a larger set.  If
<br>
we then multiply by the 10-20% estimate from the first paper then the
<br>
two figures would be pretty close at about 10000.
<br>
<p>Hal
<br>
<p>[1] <a href="http://www.nature.com/cgi-taf/DynaPage.taf?file=/nature/journal/v409/n6822/full/409928a0_fs.html">http://www.nature.com/cgi-taf/DynaPage.taf?file=/nature/journal/v409/n6822/full/409928a0_fs.html</a>,
<br>
[2] <a href="http://www.nature.com/cgi-taf/DynaPage.taf?file=/nature/journal/v409/n6822/full/409821a0_r.html&filetype=#B7">http://www.nature.com/cgi-taf/DynaPage.taf?file=/nature/journal/v409/n6822/full/409821a0_r.html&filetype=#B7</a>
<br>
<p><!-- body="end" -->
<hr noshade>
<ul>
<!-- next="start" -->
<li><strong>Next message:</strong> <a href="4046.html">hal@finney.org: "Re: Is the U.S socialist? (was Re: Neal Blaike vs Michael Lorrey)"</a>
<li><strong>Previous message:</strong> <a href="4044.html">Jerry Mitchell: "RE: Is the U.S socialist? (was Re: Neal Blaike vs Michael Lorrey)"</a>
<li><strong>Maybe in reply to:</strong> <a href="4015.html">Robin Hanson: "How many genes do we differ on?"</a>
<!-- nextthread="start" -->
<!-- reply="end" -->
<li><strong>Messages sorted by:</strong> 
<a href="date.html#4045">[ date ]</a>
<a href="index.html#4045">[ thread ]</a>
<a href="subject.html#4045">[ subject ]</a>
<a href="author.html#4045">[ author ]</a>
</ul>
<!-- trailer="footer" -->
<hr noshade>
<p>
<small>
<em>
This archive was generated by <a href="http://www.hypermail.org/">hypermail 2b30</a> 
: <em>Mon May 28 2001 - 09:56:44 MDT</em>
</em>
</small>
</body>
</html>
