<!-- received="Thu Aug 27 17:07:35 1998 MDT" -->
<!-- sent="Thu, 27 Aug 1998 16:07:31 -0700 (PDT)" -->
<!-- name="Geoff Smith" -->
<!-- email="geoffs@unixg.ubc.ca" -->
<!-- subject="Re: Point mutations" -->
<!-- id="Pine.GSO.3.95q.980827154208.23080A-100000@netinfo1.ubc.ca" -->
<!-- inreplyto="b49soii9vgy.fsf@void.nada.kth.se" -->
<!-- version=1.10, linesinbody=46 -->
<html><head><title>extropians: Re: Point mutations</title>
<meta name=author content="Geoff Smith">
<link rel=author rev=made href="mailto:geoffs@unixg.ubc.ca" title ="Geoff Smith">
</head><body>
<h1>Re: Point mutations</h1>
Geoff Smith (<i>geoffs@unixg.ubc.ca</i>)<br>
<i>Thu, 27 Aug 1998 16:07:31 -0700 (PDT)</i>
<p>
<ul>
<li> <b>Messages sorted by:</b> <a href="date.html#1922">[ date ]</a><a href="index.html#1922">[ thread ]</a><a href="subject.html#1922">[ subject ]</a><a href="author.html#1922">[ author ]</a>
<!-- next="start" -->
<li><a href="1923.html">[ Next ]</a><a href="1921.html">[ Previous ]</a>
<b>In reply to:</b> <a href="1916.html">Anders Sandberg</a>
<!-- nextthread="start" -->
</ul>
<!-- body="start" -->



<p>
On 27 Aug 1998, Anders Sandberg wrote:

<p>
<a href="1916.html#1922qlink1">&gt; Damien Broderick &lt;damien@ariel.ucs.unimelb.edu.au&gt; writes:</a><br>
<i>&gt; </i><br>
<i>&gt; &gt; Everyone knows that DNA is prey to mutations, most of them quickly edited</i><br>
<i>&gt; &gt; out by vigilant repair mechanisms, most of the rest neutral or bad news,</i><br>
<i>&gt; &gt; only a few improving the genes they happen to occur within.  But by what</i><br>
<i>&gt; &gt; chemical pathway do those mutations come about?  I'd always assumed that</i><br>
<i>&gt; &gt; most of them involved gross *deletions* of one or more nucleotide bases,</i><br>
<i>&gt; &gt; the implication being that the least dangerous deletion would knock out</i><br>
<i>&gt; &gt; three adjacent bases in one go (otherwise *all* the subsequent three-letter</i><br>
<i>&gt; &gt; codons are fouled).  However, many mutations occur at a single point.  The</i><br>
<i>&gt; &gt; classic instance is the change creating sickle cell anaemia, in which</i><br>
<i>&gt; &gt; position 6 in the 146 amino acid chain has valine instead of glutamic acid.</i><br>
<i>&gt; &gt;  Presumably this means that an adenine has been changed into a uracil (or,</i><br>
<i>&gt; &gt; in the DNA chain, into a thymine).  </i><br>
<i>&gt; &gt; </i><br>
<i>&gt; &gt; But how the hell does the equivalent of a random impact do that?</i><br>
<i>&gt; </i><br>
<i>&gt; Radiation or free radicals mess up single nucleotides, changing them</i><br>
<i>&gt; into other molecules, but I think the major contribution is errors in</i><br>
<i>&gt; replication. If a single base is damaged (or just unlucky), the enzyme</i><br>
<i>&gt; doing the replication might make a wild guess and insert a random</i><br>
<i>&gt; nucleotide in the copy.</i><br>
<i>&gt; </i><br>
<i>&gt; Just a guess from a computer scientists playing molecular biologist :-)</i><br>

<p>
Well, I'm on the road to being both, and I can verify your wild guess.
Many mutagens work by altering a nucleotide so that it binds better with
a noncomplementary nitrogenous base, giving you a weird
base-pair combination like Adenine-Guanine.  If the change made by the
mutagen is permanent, the error in pairing can be missed by
"proof-reading" enzymes, and you will end up with half the progeny being
mutants, the rest being normal.

<p>
If my memory of biochem serves me, mutations can also occur without a
mutagen from the random and rare ketone to -enol molecular change in a
nitrogenous base.  This would also create a weird base pairing, but I
think it is usually weeded out by proof-reading enzymes.

<p>
Geoff.
<!-- body="end" -->
<p>
<ul>
<!-- next="start" -->
<li><a href="1923.html">[ Next ]</a><a href="1921.html">[ Previous ]</a>
<b>In reply to:</b> <a href="1916.html">Anders Sandberg</a>
<!-- nextthread="start" -->
</ul>
</body></html>
